frappe/patches/v12_0/multi_website.py

```diff
@@ -0,0 +1,54 @@
+import frappe
```

Comment:
please add comments in code

https://github.com/frappe/erpnext/wiki/Coding-Standards#code-comments

Reply:
Will do.